.github/workflows/coverage-generator.yml

```
@@ -0,0 +1,177 @@
+    steps:
+      - name: Restore source code
+        uses: actions/cache@v4
+        with:
+          path: .
+          key: ${{ runner.os }}-source-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-source-
+
+      - name: Get the latest Jacoco report URL
+        id: get-artifact-url-oidc
+        run: |
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)